docs/reference/server-config.md

```diff
@@ -59,6 +59,12 @@ learn more about the configuration section corresponding to that key:
       "release_stage": "production"
     }
   }
+  <a href="#caching-section-optional">"caching"</a>a>: {
```

Comment:
Great Docs! 

There seems a typo, I'll create a PR for this since maybe you have already out of the office :)